eval/alpaca_farm/run_eval.py

```diff
@@ -108,7 +108,6 @@ def main(args):
         df_leaderboard, annotations = alpaca_farm_evaluate(
             model_outputs=model_results,
             reference_outputs=args.reference_path,
-            annotators_config="alpaca_eval_gpt4",
```

Comment:
So, I guess we use their default configs all the time?

Reply:
Yeah, AlpacaEval also swaps the default depending on the `IS_ALPACA_EVAL_2` flag. Our config there matches their default: https://github.com/tatsu-lab/alpaca_eval/blob/main/src/alpaca_eval/constants.py#L42